pkgs/misc/urbit/fetch-github-lfs.nix

```diff
@@ -0,0 +1,107 @@
+{ lib, stdenvNoCC, runCommandLocal, cacert, curl, jq }:
```

Comment:
We should rather expand fetchFromGitHub or make a generic top-level fetcher instead.